From 7dd258a2a68bfe4756ded42128dd377278508e5d Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Tue, 18 May 2021 17:01:41 +0800
Subject: [PATCH] arch: arm64: dts: keep i2c1 in disabled status

i2c1 pins are conflict with llce feature, so keep it in
disabled status.

Upstream-Status: Pending

Signed-off-by: Meng Li <Meng.Li@windriver.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb.dtsi b/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb.dtsi
index a050c098b8fd..a13e92df9568 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb.dtsi
@@ -90,7 +90,6 @@ &i2c1 {
 	pinctrl-1 = <&pinctrl1_i2c1_gpio>;
 	scl-gpios = <&gpio1 163 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
 	sda-gpios = <&gpio1 165 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
-	status = "okay";
 };
 
 &i2c2 {
-- 
2.25.1

